packages/wp-story-editor/src/components/checklist/accessibility/videoOptimizationCheckbox/index.js

```diff
@@ -66,44 +84,86 @@ function VideoOptimizationCheckbox() {
   const checked = currentUser?.meta?.web_stories_media_optimization;
   const showCheckbox = !checked && hasUploadMediaAction;
 
-  return showCheckbox && isChecklistMounted ? (
+  const handleToggle = useCallback(() => {
+    setHasOptedIn(true);
+    toggleWebStoriesMediaOptimization();
+  }, [toggleWebStoriesMediaOptimization]);
+
+  const { isSaving, saveStory } = useStory(({ state, actions }) => ({
+    isSaving: state.meta.isSaving,
+    saveStory: actions.saveStory,
+  }));
+  const handleSave = async () => {
+    await saveStory();
+    window.location.reload(true);
+  };
+
+  return (showCheckbox || hasOptedIn) && isChecklistMounted ? (
     <ChecklistCard
       title={__(
         'Optimize all videos in the Story to ensure smooth playback.',
         'web-stories'
       )}
       footer={
-        <CheckboxContainer>
-          <Checkbox
-            id="automatic-video-optimization-toggle"
-            checked={currentUser?.meta?.web_stories_media_optimization}
-            onChange={toggleWebStoriesMediaOptimization}
-          />
-          <CheckboxLabel
-            forwardedAs="label"
-            htmlFor="automatic-video-optimization-toggle"
-          >
-            <TranslateWithMarkup
-              mapping={{
-                a: (
-                  <Link
-                    size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-                    onClick={(evt) =>
-                      trackClick(evt, 'click_video_optimization_settings')
-                    }
-                    href={dashboardSettingsLink}
-                    isBold
-                  />
-                ),
-              }}
+        hasOptedIn ? (
+          <>
+            <p>
```

Comment:
Let's use the same `DefaultFooterText` (`Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}`) for the paragraph here so that spacing is consistent.